frontend/packages/knative-plugin/src/components/overview/__tests__/OverviewDetailsKnativeResourcesTab.spec.tsx

```diff
@@ -55,4 +58,15 @@ describe('OverviewDetailsKnativeResourcesTab', () => {
         .props().item,
     ).toEqual(knItem.item);
   });
+
+  it('should render Routes, Configuration and revision list on sidebar in case of kn deployment', () => {
+    knItem.item = {
+      ...knItem.item,
+      ...{ obj: MockKnativeResources.deployments.data[0] },
```

Comment:
nit: no need to spread
```suggestion
      obj: MockKnativeResources.deployments.data[0],
```